deploy/common/deploy.sh

```diff
@@ -108,6 +108,34 @@ function wait_for_central {
     echo
 }
 
+# Checks if central already exists in this cluster.
+# If yes, the user is asked if they want to continue. If they answer no, then the script is terminated.
+function check_central_exists() {
```

Comment:
Suggesting to make the intent more clear from the name:

```suggestion
function prompt_if_central_exists() {
```